duckduckgo/BrowserServicesKit
# Add additional parameters to breakage reports

Please review the release process for BrowserServicesKit [here](https://app.asana.com/0/1200194497630846/1200837094583426).

**Required**:

Task/Issue URL: https://app.asana.com/0/414235014887631/1206842318329792/f
iOS PR: https://github.com/duckduckgo/iOS/pull/2592
macOS PR: https://github.com/duckduckgo/macos-browser/pull/2416
What kind of version bump will this require?: Major/Minor/Patch

**Optional**:

Tech Design URL:
CC:

**Description**:



**Steps to test this PR**:
1. Search for something on DDG
1. Click a link on the SERP
2. Set a breakpoint to view the generated breakage report and submit the report
3. Ensure the `openerContext` is SERP and `vpnOn` reflects the current NetP connection status



**OS Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
